src/app/models/page.ts

```diff
@@ -5,4 +5,13 @@ export class Page extends AbstractPage {
   filename: string;
   resource: Resource;
   position: number;
+
+  static compare(a: Page, b: Page): number {
+    if (a.position < b.position) {
```

Comment:
I think you could just do `return a.position - b.position;`.

Reply:
I like this, I don't think I've ever thought of that before

Reply:
`return a.attr.localeCompare(b.attr);` is also super nice if you are dealing with strings (which you aren't in this case).